### PR TITLE
feat: support shuvi lint

### PR DESCRIPTION
**Pull request by wangjinyang**
**What kind of change does this PR introduce?**
feat: support shuvi lint

**Did you add tests(unit,e2e) for your changes?**
yes

**Does this PR introduce a breaking change?**
no

**What needs to be documented once your changes are merged?**
support shuvi lint

### CODE REVIEW

File: packages/eslint-config-shuvi/package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/shuvijs/shuvi.git",
+    "directory": "packages/eslint-config-shuvi"
+  },
+  "scripts": {
+    "test-pack": "cd ../../ && pnpm test-pack eslint-config-next"
+  },
+  "dependencies": {
+    "@babel/eslint-parser": "7.18.2",
```
Comment (liximomo) [inline packages/eslint-config-shuvi/package.json:16]: 依赖里好像少了 eslint-plugin-shuvi?

File: packages/eslint-config-shuvi/index.js [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+  //       img: ['Image']
+  //     }
+  //   ],
+  //   'jsx-a11y/aria-props': 'warn',
+  //   'jsx-a11y/aria-proptypes': 'warn',
+  //   'jsx-a11y/aria-unsupported-elements': 'warn',
+  //   'jsx-a11y/role-has-required-aria-props': 'warn',
+  //   'jsx-a11y/role-supports-aria-props': 'warn'
+  // },
+  rules: {
```
Comment (liximomo) [inline packages/eslint-config-shuvi/index.js:82]: rule部分和next不太一致，需要讨论一下

**Review by liximomo — CHANGES_REQUESTED** [review]

File: packages/eslint-config-shuvi/package.json (hunk 1 above)
Comment (wangjinyang) [reply packages/eslint-config-shuvi/package.json:16]: 在下面 "@shuvi/eslint-plugin-shuvi": "1.0.22",

File: packages/shuvi/package.json [hunk 3]
```diff
@@ -26,6 +26,11 @@
     "node": ">= 16.0.0"
   },
   "dependencies": {
+    "@babel/code-frame": "7.12.11",
```
Comment (evenchange4) [inline packages/shuvi/package.json:29]: - [ ] `"@babel/code-frame"` should be listed as a devDependencies.

**Review by evenchange4 — CHANGES_REQUESTED** [review]

File: packages/eslint-config-shuvi/index.js (hunk 2 above)
Comment (wangjinyang) [reply packages/eslint-config-shuvi/index.js:82]: 一期只要機制，二期再 review 其餘 rules

File: packages/shuvi/package.json (hunk 3 above)
Comment (wangjinyang) [reply packages/shuvi/package.json:29]: 这个不是devDependencies是dependencies

**Review by liximomo — APPROVED** [review]
/lgtm

**Pull request merged**